book/chapters/electrons.ipynb

```diff
@@ -40,64 +44,71 @@
       "text/plain": [
-       "<Figure size 432x288 with 1 Axes>"
+       "<Figure size 640x480 with 1 Axes>"
       ]
      },
-     "metadata": {
-      "needs_background": "light"
-     },
+     "metadata": {},
      "output_type": "display_data"
     }
    ],
    "source": [
-    "plt.hist(ak.flatten(electrons.pt), bins=100, range=(0, 100))\n",
+    "plt.hist(pt_data, bins=100, range=(0, 100))\n",
```

Comment:
Yeah - what was there before is much more expressive. We should try to keep that.